src/pkcs11/framework-pkcs15.c

```diff
@@ -3879,8 +3879,9 @@ pkcs15_pubkey_get_attribute(struct sc_pkcs11_session *session, void *object, CK_
 		break;
 	case CKA_TOKEN:
 	case CKA_SENSITIVE:
```

Comment:
check `git blame` to get to know who did it. But it should more usefull to check the standard...